src/pages/public_registration/action_accordions/DoiRequestAccordion.tsx

```diff
@@ -67,6 +69,8 @@ export const DoiRequestAccordion = ({
   const [messageToCurator, setMessageToCurator] = useState('');
   const [openRequestDoiModal, setOpenRequestDoiModal] = useState(false);
   const toggleRequestDoiModal = () => setOpenRequestDoiModal((open) => !open);
+  const [showConfirmDialogAssignDoi, setShowConfirmDialogAssignDoi] = useState(false);
+  const toggleConfirmDialogAssignDoi = () => setShowConfirmDialogAssignDoi((open) => !open);
```

Comment:
Noen spesiell grunn til at du har laget en egen funksjon for dette, istede for å bare sette state direkte?

Reply:
Brukes tre plasser, så synes det var greit med gjenbruk :)